zerver/tests/tests.py

```diff
@@ -201,19 +219,23 @@ def test_admin_api(self):
 class WorkerTest(TestCase):
     class FakeClient(object):
         def __init__(self):
-            self.consumers = {}
-            self.queue = []
+            # type: () -> None
+            self.consumers = {} # type: Dict[str, Callable]
+            self.queue = [] # type: List[Dict[str, Any]]
```

Comment:
`self.queue` should be `List[Tuple[text_type, Dict[str, Any]]]` (or maybe `str` instead of `text_type`). I deduced this using [line R253](https://github.com/zulip/zulip/pull/1043/commits/594eda27a1b4ae437eacabd752aac35f8a108709#diff-3eec6d38a1395d020c53c58e6efed31dR253).